skbio/sequence/_sequence.py

```diff
@@ -887,7 +933,22 @@ def __str__(self):
         return str(self._string.decode("ascii"))
 
     def __repr__(self):
-        """Return a string representation of the biological sequence object.
+        r"""Return a string representation of the biological sequence object.
```

Comment:
Seems like this docstring would be valuable information for users, but now that we're not putting `__` methods in our docs they'll never see it, right?